dev/src/view/TreeItemContext.ts

```diff
@@ -138,7 +138,7 @@ namespace TreeItemContext {
             contextValues.push(TreeItemContextValues.PROJ_RESTARTABLE);
         }
 
-        if (project.capabilities.metricsAvailable) {
+        if (project.capabilities.metricsAvailable || project.language === 'java') {
```

Comment:
for reviewers: this PR also gives more project types the `open app monitor` button The additional projects that will have the button are java projects that are not microprofile or spring (e.g. Open Liberty, which is type `docker`). 

(Since `metricsAvailable` will always be `false` for them, clicking on `open app monitor` will always open the app monitor hosted on our performance container)